modules/hcp-bm-autoscale.adoc

```diff
@@ -99,3 +99,10 @@ $ oc --kubeconfig ./hostedcluster-secrets -n <namespace> delete deployment <depl
 ----
 $ oc --kubeconfig ./hostedcluster-secrets get nodes
 ----
+
+[NOTE]
+====
+For {ibm-z-title} agents, compute nodes get detached from the cluster only for {ibm-z-title} with KVM agents. For z/VM and LPAR, you must delete the compute nodes manually.
+
+Agents can be reused only in case of {ibm-z-title} with KVM, for z/VM and LPAR, recreate the agents for using as compute nodes
```

Comment:
```suggestion
Agents can be reused only for {ibm-z-title} with KVM. For z/VM and LPAR, re-create the agents to use them as compute nodes.
```
Check my revision for accuracy. I wasn't entirely sure what "recreate the agents for using as compute nodes" meant.